src/arrayMethodSort.js

```diff
@@ -3,9 +3,21 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const value = this[i];
+
+          this[i] = this[j];
+          this[j] = value;
+        }
+      }
+    }
```

Comment:
This implementation of the sort method will mutate the original array. If this is intentional, it's fine; however, it's worth noting that some sort implementations return a new sorted array without mutating the original one.